<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pagePointListWgt</class>
 <widget class="QWidget" name="pagePointListWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>pagePointListWgt</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="wgt_top" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#wgt_top{
background:#203B73;
border-radius:4px 4px 0px 0px;
}
QLabel{
font-family:&quot;微软雅黑&quot;;
font-size:16px;
color:#FFFFFF;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>交班点货单</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wgt_bottom" native="true">
     <property name="styleSheet">
      <string notr="true">#wgt_bottom{
background:#FFFFFF;
border-radius:0px 0px 4px 4px;
}
#btn_next{
font-family:&quot;微软雅黑&quot;;
font-size:14px;
color:#FFFFFF;
background:#00B900;
border-radius:4px;
}
#btn_next:pressed{
margin:1px;
}
#checkBox{
font-family:&quot;微软雅黑&quot;;
font-size:14px;
color:rgba(0,0,0,0.9);
}
#checkBox::indicator{
width:14px;
height:14px;
}
#checkBox::indicator:unchecked{
image:url(:/masterCashier/Resources/check_common.png);
}
#checkBox::indicator:checked{
image:url(:/masterCashier/Resources/check_select.png);
}
#wgt_search{
background-color:#F8F8F8;
border-top-left-radius:4px;
border-top-right-radius:4px;
}
#btn_search{
border-image:url(:/masterCashier/Resources/search.png);
border:none;
}
#lineEdit{
border:none;
font-family:&quot;微软雅黑&quot;;
font-size:14px;
color:#000000;
background:#F8F8F8;
}
#wgt_list{
border:1px solid #DEEEF2;
border-radius:2px;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QWidget" name="wgt_search" native="true">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>40</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="leftMargin">
            <number>20</number>
           </property>
           <property name="rightMargin">
            <number>20</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="btn_search">
               <property name="minimumSize">
                <size>
                 <width>18</width>
                 <height>18</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>18</width>
                 <height>18</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit">
               <property name="placeholderText">
                <string>检索商品/扫描条码</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>只查看销售数据</string>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="wgt_list" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>488</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#listWidget{
border:1px solid #DEEEF2;
border-top:none;
border-bottom:none;
}
#wgt_header{
background:#edfbfe;
border:1px solid #DEEEF2;
border-bottom:none;
border-radius:2px 2px 0px 0px;
}
#wgt_page{
/*border:1px solid #DEEEF2;*/
border-top:none;
border-radius:0px 0px 2px 2px;
}
</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="wgt_header" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>32</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
font-family:&quot;微软雅黑&quot;;
font-size:14px;
color:rgba(0,0,0,0.9);
}</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>20</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>序号</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="minimumSize">
               <size>
                <width>190</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>190</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>商品名称</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="minimumSize">
               <size>
                <width>92</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>92</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>类别</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>零售单位</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="minimumSize">
               <size>
                <width>114</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>114</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>零售价(元)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>仓库数量</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>交班数量</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget">
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="PageWidget" name="wgt_page" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>42</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>42</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
font-family:&quot;微软雅黑&quot;;
font-size:14px;
color:#656565;
}
QLineEdit{
font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#3A3A3A;
border:1px solid rgba(0,0,0,0.05);
background:#F5F5F5;
border-radius:1px;
}
QPushButton{
font-family:&quot;微软雅黑&quot;;
font-size:12px;
}
#btn_previous_page{
border-image:url(:/masterCashier/Resources/page_front.png);
border:none;
}
#btn_previous_page:pressed{
margin:1px;
}
#btn_next_page{
border-image:url(:/masterCashier/Resources/page_next.png);
border:none;
}
#btn_next_page:pressed{
margin:1px;
}
</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btn_next">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>下一步</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PageWidget</class>
   <extends>QWidget</extends>
   <header>PageWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
